Basic-Audio-Transformer/js/transformer.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-unused-vars */
+/* global Float32Array AudioData */
+
+export default class AudioTransformer {
```

Comment:
How do you feel about giving folks a tl;dr of what this transformer does to audio?

Reply:
Nit: it's odd that "lowpass" is in quotes.  It's an actual thing :-)